### PR TITLE
do not parse html quotations if html is longer then certain threshold

### PR DESCRIPTION
**Pull request by obukhov-sergey**

### CODE REVIEW

**Comment by obukhov-sergey** [comment]
@mailgun-ci test this please

**Comment by obukhov-sergey** [comment]
@mailgun-ci test this please

File: talon/quotations.py [hunk 1]
```diff
@@ -164,6 +164,7 @@
 
 SPLITTER_MAX_LINES = 4
 MAX_LINES_COUNT = 1000
+MAX_HTML_LEN = 2794202
```
Comment (anton-efimenko) [inline talon/quotations.py]: I would add a comment how you came up with this number.

**Comment by anton-efimenko** [comment]
lgtm!

**Pull request merged**